app/src/main/java/org/mozilla/rocket/home/HomeViewModel.kt

```diff
@@ -93,6 +94,7 @@ class HomeViewModel(
     val isFxAccount: LiveData<Boolean> = getIsFxAccountUseCase()
     val shouldShowNewMenuItemHint: LiveData<Boolean> = shouldShowNewMenuItemHintUseCase()
     val isContentHubEnabled: LiveData<Boolean> = shouldShowContentHubUseCase()
+    val homeBackgroundColorTheme = MutableLiveData<ThemeManager.ThemeSet>()
```

Comment:
we should probably initialize the value at the beginning. But we might able to do it in the future.